src/Addon/Plugin/PluginCriteria.php

```diff
@@ -88,7 +81,17 @@ public function cache($parameters = null)
             ];
         }
 
-        $this->options['cache'] = $parameters;
+        array_set(
+            $this->options,
+            'cache',
+            array_merge(
```

Comment:
This has to be `array_merge` to NOT overwrite cache options that had been set by `$this->setCachePrefix()` or `$this->setCacheTtl()` before.